docs/changelog.md

```diff
@@ -1,8 +1,31 @@
 # Change Log
 
-## Version 2.0 Alpha 3 (2019-07-04)
+## Version 2.0 Beta 1 (2019-17-30)
```

Comment:
@pyricau, I think you meant `07` instead of `17`.

Reply:
Thanks!